.env.example

```diff
@@ -1,4 +1,4 @@
 NEXT_PUBLIC_INFURA_ID=__ID__
 NEXT_PUBLIC_MAINNET_URL=__URL__
 NEXT_PUBLIC_GOERLI_URL=__URL__
-NEXT_PUBLIC_AUTONOLAS_URL=__URL__
\ No newline at end of file
+NEXT_PUBLIC_GOVERNANCE_URL=__URL__
```

Comment:
I've left a bunch of comments about [issues on the experience side](https://docs.google.com/document/d/1IU_KtuOK-dOFCzbbXqYfJOaFobZjvUGeO-rDnbqA1e4/edit#). Want to resolve those before looking at the code.